### PR TITLE
Added customerData views

### PR DESCRIPTION
**Pull request by alexey-rise**
@tejohnso @andrecardoso please review.

### CODE REVIEW

File: projects/Rise_Stats/datasets/customerData/views/companySnapshot.bq [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+select 
+  C.companyId,
+  C.name,
+  C.parentId,
+  C.street, 
+  C.unit,
+  C.city,
+  C.province as state,
+  C.country,
+  C.creationDate
```
Comment (andrecardoso) [inline projects/Rise_Stats/datasets/customerData/views/companySnapshot.bq]: Can you please include `C.hubSpotId` here?

File: projects/Rise_Stats/datasets/customerData/views/userSnapshot.bq [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  U1.userId,
+  firstName,
+  lastName,
+  roles as role, 
+  email as contactEmail,
+  creationDate, 
+  lastLogin,
+  mailSyncEnabled as subscribedToNewsletter,
+  U3.subscriptionDate,
+  if(mailSyncEnabled = false, U4.unsubscriptionDate, null) as unsubscriptionDate
```
Comment (andrecardoso) [inline projects/Rise_Stats/datasets/customerData/views/userSnapshot.bq]: Please include `hubSpotId` here

File: projects/Rise_Stats/datasets/customerData/views/companySnapshot.bq (hunk 1 above)
Comment (alexey-rise) [reply projects/Rise_Stats/datasets/customerData/views/companySnapshot.bq]: @andrecardoso added.

File: projects/Rise_Stats/datasets/customerData/views/userSnapshot.bq (hunk 2 above)
Comment (alexey-rise) [reply projects/Rise_Stats/datasets/customerData/views/userSnapshot.bq]: @andrecardoso added.

**Pull request merged**